README.md

```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# Homework 01
+
+Hello world! My name is Marc-Antoine Manzoni.
+I'm a second (soon to be third year) in *The College*.
+I am majoring in History, and minoring in East Asian Languages and Cultures (EALC). Please enjoy this picture of me and a very nice dog I met recently.
+![this picture](Images/4fc3f091-bd9b-4edb-9f5a-34505d985046.JPG). 
```

Comment:
I'm glad you were able to figure out how to upload the photo! Just a super quick note: the period at the end of this line (line 6) shows up after the photo - not a huge deal at all, but just wanted to let you know!